calico_cni/calico_cni.py

```diff
@@ -336,14 +357,18 @@ def _create_endpoint(self, assigned_ip):
                                                     self.container_id,
                                                     [assigned_ip])
         except AddrFormatError:
```

Comment:
We probably still want to log an exception here, since it will help debugging.
```
_log.exception("Failed to create Calico endpoint")
```

I'm not sure how I feel about logging an error in the _print_error_message(...) method, since it isn't a flexible as handling the logging in each individual failure branch.